src/actions/guides/deploying/heroku.cr

```diff
@@ -156,9 +154,11 @@ class Guides::Deploying::Heroku < GuideAction
     $ heroku config:set SEND_GRID_KEY=<KEY>
     ```
 
-    If you don't need to send emails, you'll need to change the config in
-    `config/email.cr` to use the `Carbon::DevAdapter`. When you open the file there
-    will be comments showing you what to do.
+    If you don't need to send emails you can set the value to `unused`.
+    
+    Alternatively, you can change the config in `config/email.cr` to use the
+    `Carbon::DevAdapter`. When you open the file there
+    will be comments showing you how to do this.
```

Comment:
Maybe we can also link to the email docs? https://www.luckyframework.org/guides/emails/sending-emails-with-carbon

I would imagine we will beef up that page as well, but having a link here now will help later.